### ISSUE TITLE
Update Cancel Purchase Page to Improve Clarity on Refunds and Introduce Payment Flexibility

### ISSUE THREAD

**Issue by simranvijwani** [Automattic/wp-calypso#88979]
### What

Implement two key enhancements to the Cancel Purchase page:
1. **Clear Refund Eligibility Information:** Display an explicit notice about the refund eligibility status, including the specific cutoff date and a link to the Refund Policy support page.
2. **Monthly Payment Option Prompt:** Prominently feature an option for users to switch to monthly payments, including a direct link or contact option for support, to provide users with flexible payment alternatives.

**Expected Outcomes:** 
- A significant decrease in cancellation and refund requests due to better-informed decisions by users.
- An increase in users opting for plan modifications over cancellations, thereby preserving revenue and engagement.
- A reduction in chargebacks and associated fees by setting clear expectations on refund eligibility.
- Enhanced user satisfaction through transparent communication and flexible payment options.

### Why

Many users find themselves confused about their eligibility for refunds upon initiating a cancellation. This confusion often leads to an increased volume of support tickets related to cancellations and refunds, higher instances of chargebacks, and diminished customer satisfaction. Furthermore, users lacking awareness of alternative payment options may opt for cancellation over more suitable solutions.

**Comment by Aurorum** [comment]
I love this. :) I think a big reason for this is that the action verb has always been “Cancel plan” rather than “Refund plan”: https://wordpress.com/support/refunds/

I’ve seen many users confused about this in Support, and I’d love for that to change too! But I do wonder if this was an intentional marketing decision to prevent refunds?

Happy to work on this, but would love to get a sense on whether this has definitely been approved.

**Comment by rickmgithub** [comment]
I've pinged for more on this here 

> C042CJ2NH/p1711614381966119-slack-triage

**Comment by Aurorum** [comment]
Thank you!! I don't have access to that channel on Slack so would be v grateful if you could share their response here when you receive it. :)

**Comment by michaelpick** [comment]
Hi! Just to make sure we're on the same page would you mind sharing a direct link, screengrab, or copy dump for the screen we'd like to modify?

**Comment by Aurorum** [comment]
@michaelpick these are my views (so @simranvijwani, please feel free to correct me if you're referring to something else!), I would like to change this "Cancel plan" button to "Refund plan": https://wordpress.com/support/refunds/#how-to-get-a-refund

And maybe have some sub-text just beneath it stating when the refund window ends for that purchase. 

<img width="649" alt="Screenshot 2024-04-03 at 11 01 05" src="https://github.com/Automattic/wp-calypso/assets/43215253/4d5f7dcf-786b-4eac-a0c1-324bf2ea0e67">

Can be accessed by Me > Purchases and then clicking an individual subscription

**Comment by simranvijwani** [comment]
@Aurorum Yes! The issue involves two feature requests. The first request is to add refund eligibility details as a subtext, which would include a link to the refund policy support guide.

![image](https://github.com/Automattic/wp-calypso/assets/34896144/7eebe93d-0790-4d3a-ac85-2e238dd6ed00)

The second request is to add a new button that allows users to switch to the monthly payment option.

![image](https://github.com/Automattic/wp-calypso/assets/34896144/4457867e-6e6a-4ceb-a2cc-0667d6ea2055)

**Comment by michaelpick** [comment]
Thanks, both! We'll share something ASAP!

**Comment by alvezfrancese** [comment]
Hi there! I don’t think we should change ‘Cancel plan’ to ‘Refund plan’, since it would obfuscate the real action the user's taking, which is actually canceling the plan, with the refund being a secondary thing. 

We do could, nevertheless, add some subcopy to explain about refunds (the 'Learn more' part being a link to the refund policy docs):

[Cancel plan] Get a full refund until Apr 24, 2024. Learn more.

Regarding the new button: [Switch to monthly billing] may work. 

Should we also add a [Switch to annual billing and save] CTA for monthly payers?

**Comment by Aurorum** [comment]
Cool, thanks very much! I'd like to split the two issues if that's okay, but I'll take a look at adding the refund text tomorrow.

**Comment by Aurorum** [comment]
I've proposed adding the refund text in #89249. :) 

It's not possible to include `Learn more` as a link because the Card is a link (can't have a link within another linK!), so I've added a tooltip like this instead.

<img width="1085" alt="Screenshot 2024-04-05 at 12 21 57" src="https://github.com/Automattic/wp-calypso/assets/43215253/7f0a1c3e-d644-4a21-b68f-75c2d2317040">

For context on what this looks like - let me know if you have any thoughts!

<img width="1009" alt="Screenshot 2024-04-05 at 12 14 44" src="https://github.com/Automattic/wp-calypso/assets/43215253/7074ce0f-4101-4c9b-ba3a-9877abf00ac4">

**Comment by alvezfrancese** [comment]
Hi! A tooltip is a good solution, if we need that information. What about saying something like: 'Questions? Check our refund policy', being 'refund policy' the link?

For the domain part: what do you think of 'If you're unsure of this domain, cancel it before 8 April, 2024 to choose another free one instead'?

**Comment by Aurorum** [comment]
Great shout - thank you! I've switched it to that language instead.

**Comment by DavidRothstein** [comment]
>I think a big reason for this is that the action verb has always been “Cancel plan” rather than “Refund plan"

Not quite always -- it used to be more like "Cancel and refund plan", before it was changed here: https://github.com/Automattic/wp-calypso/pull/68088

**Comment by liviopv** [comment]
📌 ACTIONS
- Feature request kept

Keeping this request. There's an ongoing project to improve this flow: pfuQfP-c9-p2 
cc @bohdankit07 in case the discussion here it's relevant to your project

**Comment by sirbrillig** [comment]
https://github.com/Automattic/wp-calypso/pull/89249 was merged and handles at least some of this.
